Removed outdated content and put in bullet point about FGAC. (#399)
momentohq/public-dev-docs: docs/develop/basics/01-working-with-momento-auth-tokens.md

Momento auth tokens have an associated `TokenScope` which controls their level of access to Momento resources. Here is a list of the available `TokenScope`s:

> - `SuperUser`: these tokens have full access to all control plane and data plane operations. They can also be used to generate new tokens via the [`generateAuthToken`](/develop/api-reference#generateauthtoken) API. Creating a `SuperUser` token must be performed in the [Momento web console](https://console.gomomento.com).

Reviewer: ```suggestion
- `SuperUser`: This token scope has full access to all control plane and data plane operations. A token with `SuperUser` scope can also be used to generate new tokens via the [`generateAuthToken`](/develop/api-reference#generateauthtoken) API. A token with `SuperUser` scope can only be created using the [Momento web console](https://console.gomomento.com).
```

> - Fine-grained access control (FGAC): These tokens enable you to more precisely control the access granted by a token. You can control if the token grants access to one or more caches and/or topics. FGAC tokens can only be used for data plane operations, but can be created in the [Momento web console](https://console.gomomento.com), or via the [Momento auth API](/develop/api-reference/auth-tokens.md).

Reviewer: ```suggestion
- Fine-grained access control (FGAC): This token scope enables you to more precisely control the access granted by a token. You can control if the token grants access to one or more caches and/or topics. Tokens with FGAC scope can only be used for data plane operations, but can be created in the [Momento web console](https://console.gomomento.com), or via the [Momento auth API](/develop/api-reference/auth-tokens.md).
```
